### DISCUSSION TITLE
Spinner are too slow with relax

### DISCUSSION THREAD

**Discussion by Wuteva** [ppy/osu#18614] [Q&A]
### Type

Game behaviour

### Bug description

Spinners used to be easier when playing with the relax mod, reaching max speed easily. This is not the case anymore and sometime it's almost impossible to not get a miss, especialy on spinners that don't last long. See the video where I play this song: https://osu.ppy.sh/beatmapsets/36323#osu/117379 on relax with lazer first and stable right after. I'm moving my tablet pen at the same speed in both case. I forgot to show the fps in the video but it does not drop so it does not seems lag related.

Also, is the spinner giving a different score (+50 instead of +1000) another bug or did the scoring system change?

### Screenshots or videos

https://user-images.githubusercontent.com/1039595/172507849-16c1944b-535a-4c9c-8a21-82715a70b17b.mp4

### Version

2022.605.1-lazer

### Logs

[runtime.log](https://github.com/ppy/osu/files/8857361/runtime.log)
[updater.log](https://github.com/ppy/osu/files/8857362/updater.log)
[database.log](https://github.com/ppy/osu/files/8857363/database.log)
[network.log](https://github.com/ppy/osu/files/8857364/network.log)
[performance.log](https://github.com/ppy/osu/files/8857365/performance.log)

**Comment by smoogipoo** [comment]
Yes, spinners in osu!stable and osu!lazer behave quite differently in regards to acceleration and movement. Scoring is also different. Both are intentional.

**Comment by peppy** [comment]
One thing of note is that as the user says, relax makes spinners easier on stable by quite some degree. I do wonder if the difference on lazer is not significant enough and needs further tweaking for relax specifically?

**Comment by smoogipoo** [comment]
Stable has a 4x multiplier on spinner acceleration. Lazer doesn't even have acceleration. From a more competitive standpoint since RX is going to be ranked, I don't think I'd want the acceleration anyway.

**Comment by peppy** [comment]
That's one argument. But relax is supposed to be for... relaxing, not competition. I'd rather keep that virtue true..

**Comment by PercyDan54** [comment]
> One thing of note is that as the user says, relax makes spinners easier on stable by quite some degree. I do wonder if the difference on lazer is not significant enough and needs further tweaking for relax specifically?

In stable relax disables the combo score bonus, but lazer doesn't. And you can just easily get #1 on a mp lobby with relax on due to its score being the same with NoMod. The score should be nerfed somehow

**Comment by Wuteva** [comment]
I would prefer that the spinner animation speed match stable, even if the spinner is not actually easier to spin. It's quite disturbing when playing with a skinned spinner that have a "needle" or other reference point like I have in the video. It feels laggy even though it's not.

**Reply by frenzibyte** [reply]
Spinners will inevitably have inertia and "match stable", see https://github.com/ppy/osu/issues/8224.
